public/pages/ReviewAndCreate/components/DetectorDefinitionFields/DetectorDefinitionFields.tsx

```diff
@@ -217,6 +217,24 @@ export const DetectorDefinitionFields = (
             description={get(props, 'detector.resultIndex', '-')}
           />
         </EuiFlexItem>
+        <EuiFlexItem>
+          <ConfigCell
+            title="Custom result index min age"
+            description={get(props, 'detector.resultIndexMinAge', '-')}
```

Comment:
Can you add unit in the following 3 items?

Reply:
updated